signing-jobs/build/sign-artifacts/umb_producer.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python
+import base64
+import datetime
+import json
+import logging
+import ssl
+import subprocess
+import sys
+
+import click
+import requests
+from rhmsg.activemq.producer import AMQProducer
+
+# Expose errors during signing for debugging
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+######################################################################
+URLS = {
+    'dev': (
+        'amqps://messaging-devops-broker01.dev1.ext.devlab.redhat.com:5671',
+        'amqps://messaging-devops-broker02.dev1.ext.devlab.redhat.com:5671',
+    ),
+    'qa': (
+        'amqps://messaging-devops-broker01.web.qa.ext.phx1.redhat.com:5671',
+        'amqps://messaging-devops-broker02.web.qa.ext.phx1.redhat.com:5671',
+    ),
+    'stage': (
+        'amqps://messaging-devops-broker01.web.stage.ext.phx2.redhat.com:5671',
+        'amqps://messaging-devops-broker02.web.stage.ext.phx2.redhat.com:5671',
+    ),
+    'prod': (
+        'amqps://messaging-devops-broker01.web.prod.ext.phx2.redhat.com:5671',
+        'amqps://messaging-devops-broker02.web.prod.ext.phx2.redhat.com:5671',
+    ),
+
+}
+
+TOPIC = 'VirtualTopic.eng.art.artifact.sign'
+
+# TODO: In the future we need to handle 'rhcos' having '4.1'
+# hard-coded into the URL path.
```

Comment:
Huh? What exactly needs to handle that?
BTW I should mention. The layout in which we've delivered RHCOS in the past is not the way Clayton wants it. See https://jira.coreos.com/browse/ART-600 for how it's supposed to be at GA. We might want to practice doing it now...

Reply:
I do not plan on addressing that in this PR.